### PR TITLE
Simple script to prefill a changelog entry

### PR DESCRIPTION
**Pull request by SkySkimmer**

### CODE REVIEW

File: dev/tools/make-changelog.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+where=$(echo doc/changelog/"$where"*)
+where="$where/$PR-$(git rev-parse --abbrev-ref HEAD).rst"
+
+# shellcheck disable=SC2016
+# the ` are regular strings, this is intended
+# use %s for the leading - to avoid looking like an option (not sure
+# if necessary but doesn't hurt)
+printf '%s bla bla (`#%s <https://github.com/coq/coq/pull/%s>`_, by %s).' - "$PR" "$PR" "$(git config user.name)" > "$where"
+
+$(git config core.editor) "$where" || echo "$where"
```
Comment (Zimmi48) [inline dev/tools/make-changelog.sh]: `git config core.editor` returns an empty value for me (you could use the `$EDITOR` environment variable which is defined for me as a fallback). In any case, this line is not very robust to `git config core.editor` returning an empty value: the fact that you get a strange error message is confusing.

**Review by Zimmi48 — APPROVED** [review]
Very good! Please add code owners for this script (could be you as primary and me as secondary).

**Comment by SkySkimmer** [comment]
Updated

**Review by Zimmi48 — APPROVED** [review]
I did not retested it but it LGTM.

**Comment by Zimmi48** [comment]
I forgot to request documentation. I will add some in a further PR.

**Pull request merged**